### PR TITLE
Adding fastapi integration docs

### PR DESCRIPTION
**Pull request by sebassdc**
## Description

Adding FastAPI integration to the docs

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x ] Documentation

## Checklist

- [x ] My code follows the code style of this project.
- [ x] My change requires a change to the documentation.

### CODE REVIEW

**Comment by botberry** [comment]
Hi, thanks for contributing to Strawberry 🍓!

We noticed that this PR is missing a `RELEASE.md` file. We use that to automatically do releases here on GitHub and, most importantly, to PyPI!

So as soon as this PR is merged, a release will be made 🚀.

Here's an example of `RELEASE.md`:

```markdown
Release type: patch

Description of the changes, ideally with some examples, if adding a new feature.
```

Release type can be one of patch, minor or major. We use [semver](https://semver.org/), so make sure to pick the appropriate type. If in doubt feel free to ask :)

File: docs/integrations/fastapi.md
Comment (patrick91) [inline docs/integrations/fastapi.md:7]:
```suggestion
Strawberry provides support for [FastAPI](https://fastapi.tiangolo.com/) with the ASGI integration.

See below example for integrating FastAPI with Strawberry:
```

File: docs/integrations/fastapi.md
Comment (patrick91) [inline docs/integrations/fastapi.md:21]:
```suggestion
For more information about Strawberry ASGI refer to [asgi.md](./asgi.md)
```

**Review by patrick91 — COMMENTED** [review]
Awesome, thanks for taking time to write this PR :)

Can you also add a link in the readme? https://github.com/strawberry-graphql/strawberry/blob/main/docs/README.md

File: docs/integrations/fastapi.md
Comment (sebassdc) [reply docs/integrations/fastapi.md]: @patrick91 Added!

File: docs/integrations/fastapi.md
Comment (sebassdc) [reply docs/integrations/fastapi.md]: @patrick91 added!

**Review by patrick91 — APPROVED** [review]
@sebassdc thank you! I'll use this PR to test our docs preview for PRs! hope that's ok, will merge when that's done :)

In the meanwhile I'll invite you to the organisation and if you want a sticker feel free to fill this form: https://forms.gle/dmnfQUPoY5gZbVT67

**Comment by patrick91** [comment]
Sorry didn't get time to improve the GitHub action! Will merge this now 😊

File: docs/integrations/fastapi.md
Comment (patrick91) [inline docs/integrations/fastapi.md:16]:
```suggestion
graphql_app = GraphQL(schema)
```

File: docs/integrations/fastapi.md
Comment (patrick91) [inline docs/integrations/fastapi.md:19]:
```suggestion
app.add_route("/graphql", graphql_app)
```

**Review by patrick91 — APPROVED** [review]

File: docs/integrations/fastapi.md
Comment (patrick91) [inline docs/integrations/fastapi.md:9]:
```suggestion
See below example for integrating FastAPI with Strawberry:

```

**Pull request merged**
